tests/common/devices/sonic.py

```diff
@@ -1365,6 +1365,48 @@ def get_feature_status(self):
             feature_status[r[0]] = r[1]
         return feature_status, True
 
+    def aspath_config(self, as_path, bgp_num, counter):
```

Comment:
The functions added to `sonic.py` are not generic enough to me. Currently they are only used by this new test.
Probably these functions should just stay in the test script and do not go to the `sonic.py` file?